providers/shared/components/user/id.ftl

```diff
@@ -32,18 +32,20 @@
                 "Children" : [
                     {
                         "Names" : "Formats",
+                        "Description" : "The type of credentials to generate - system provides api level access, console is user level access",
                         "Types" : ARRAY_OF_STRING_TYPE,
                         "Values" : ["system", "console"],
                         "Default"  : [ "system" ]
                     }
                     {
                         "Names" : "EncryptionScheme",
+                        "Description" : "A prefix added to the start of the encrypted value to show that it is encrypted",
                         "Types" : STRING_TYPE,
-                        "Values" : ["base64"],
```

Comment:
This was based on the support elsewhere for encrypting values. I guess we can still do that but this allows users to roll their own if they wish?